src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -254,9 +200,6 @@ namespace OpenLoco
         Input::init();
         Input::initMouse();
 
-        // rain-related
-        _52339C = -1;
```

Comment:
We're removing the calls to its only users in this PR:
<img width="271" alt="Screenshot 2024-07-23 at 22 44 35" src="https://github.com/user-attachments/assets/e4f95ea5-0d46-4965-980f-af4391497d39">